products/amazon-rds-mysql.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Amazon RDS for MySQL
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+
+releases:
+-   releaseCycle: "8.0"
+    releaseDate: 2018-10-23
+    eol: 2026-04-01
+    latest: "8.0.32"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.7"
+    releaseDate: 2016-02-22
+    eol: 2023-10-01
+    latest: "5.7.41"
+    latestReleaseDate: 2023-02-07
+
+-   releaseCycle: "5.6"
+    releaseDate: 2013-07-01
+    eol: 2022-03-01
+    latest: "unknown"
```

Comment:
Note for myself : check if the `unknown` value (I have added) can be problematic.

Reply:
At least in the preview it is giving it a crossed out annotation. So there appears to be some kind of handling for the word "unknown"

Reply:
It does not seem to be problematic for now. Let's use it and change it if it becomes problematic.